prow/plugins/config.go

```diff
@@ -1786,9 +1786,27 @@ func (c *Configuration) mergeFrom(other *Configuration) error {
 		errs = append(errs, fmt.Errorf("failed to merge .bugzilla from supplemental config: %w", err))
 	}
 
+	if err := c.mergeApproveFrom(other.Approve); err != nil {
+		errs = append(errs, fmt.Errorf("failed to merge .approve from supplemental config: %w", err))
+	}
+
 	return utilerrors.NewAggregate(errs)
 }
 
+func (c *Configuration) mergeApproveFrom(other []Approve) error {
```

Comment:
This never returns an error and just making it `append` without the two additional conditions might be simpler :)